src/main/java/com/machine_coding/snakes_and_ladders/entities/Board.java

```diff
@@ -0,0 +1,64 @@
+package com.machine_coding.snakes_and_ladders.entities;
+
+public class Board extends Entity{
+
+    private int size;
+    private Snake[] snakes;
+    private Ladder[] ladders;
+    private Player[] players;
+    private int personPoint;
```

Comment:
what does personPoint mean?

Reply:
Changed it to `currentPlayer` as it will be used to get the which player out of the Player array has the current chance.